vpr/src/pack/cluster.cpp

```diff
@@ -633,6 +639,9 @@ std::map<t_logical_block_type_ptr, size_t> do_clustering(const t_packer_opts& pa
                                      next_molecule->pack_pattern->name, next_molecule->atom_block_ids.size());
                             VTR_LOG("\n");
                             fflush(stdout);
+                        } else if (block_pack_status == BLK_FAILED_FLOORPLANNING) {
+                            VTR_LOG("\tFloorplanning constraints not met: '%s' (%s)", blk_name.c_str(), blk_model->name);
```

Comment:
if you add the index (for using in debugger) of the atom, add for the other pack statuses as well. If not needed, don't add it though